libnavigation-ui/build.gradle

```diff
@@ -63,8 +63,8 @@ dependencies {
   implementation project(':libnavigation-util')
 
   // Mapbox
+  api dependenciesList.mapboxSdkTurf
```

Comment:
why did `Turf` dependency stay `api`?

Reply:
Examples is using `Turf` related constants. Hence it was changed from `implementation` to `api`

Reply:
In the legacy, we're exposing `Turf` in `libandroid-navigation` https://github.com/mapbox/mapbox-navigation-android/blob/fa2e50c2661bfbfacdad14bd8f3c967237beb34f/libandroid-navigation/build.gradle#L60 and not anymore 👀 https://github.com/mapbox/mapbox-navigation-android/blob/fa2e50c2661bfbfacdad14bd8f3c967237beb34f/libnavigation-core/build.gradle#L40 Do we want to do the same so this dependency wouldn't be needed in `libnavigation-ui` at all?

Reply:
Yeah we can change the `implementation` to `api` in `libnavigation-core/build.gradle`